_episodes_rmd/13-dplyr.Rmd

```diff
@@ -1,9 +1,9 @@
 ---
-title: Manipulación de **Dataframes** con dplyr
+title: Manipulación de **dataframes** con dplyr
```

Comment:
The bold formatting needs to be removed because this does not work in titles. Instead the ** remain in the title

Reply:
yeah data frames should be two words, 
The reason why it was **dataframe** its because its a type of data in R and we wrote in the conventions not to translate data types and put them in bold because they are used as original.
(for example **integer** should not be entero)

I agree for the title to use is as two words not bold

Reply:
Ah yes, I often write **dataframe**, so I'm sure the original lesson was written that way, but looking through official R documentation, **data frame** seems to be correct.